src/main.rs

```diff
@@ -20,6 +20,8 @@ pub struct Args {
     pub dir: Option<String>,
     #[arg(short, long, help = "Namespace to filter [default: '']")]
     pub namespace: Option<String>,
+    #[arg(short, long, help = "Verbose output")]
+    pub verbose: bool,
```

Comment:
im wondering if instead I should call this `context` ? since maybe we might want to add actual logging that would be toggled via this flag in the future?

Reply:
I like that idea